tests/integration/query/simple/with_fragments_test.go

```diff
@@ -0,0 +1,256 @@
+// Copyright 2024 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package simple
+
+import (
+	"testing"
+
+	testUtils "github.com/sourcenetwork/defradb/tests/integration"
+)
+
+func TestQuerySimpleWithFragmentsSucceeds(t *testing.T) {
```

Comment:
suggestion: I'm not sure if you had changed them last time it was suggested but `TestQuerySimple_WithFragments_Succeeds` would read a little better and would be easier to go straight to the most descriptive part (`WithFragment`) of the test name.

Reply:
done